test/parallel-benchmark/mzcompose.py

```diff
@@ -263,7 +268,8 @@ def run(
         measurements: collections.defaultdict[str, list[Measurement]],
     ) -> None:
         for _ in self.dist.generate(duration):
-            jobs.put(lambda: self.action.run(conns, measurements))
+            start_time = time.time()
+            jobs.put(lambda: self.action.run(start_time, conns, measurements))
```

Comment:
We should even let `generate` yield the start time instead of capturing it here.